formats/finder/frontend/examples/policies.json

```diff
@@ -0,0 +1,37 @@
+{
+   "base_path": "/government/policies",
+   "format":"finder",
+   "title":"Policies",
+   "description":"",
+   "updated_at": "2015-02-20T10:31:55.529Z",
+   "public_updated_at":"2015-03-09T14:09:25.000+00:00",
+   "locale":"en",
+   "details":{
+      "document_noun":"policy",
+      "email_signup_enabled":false,
+      "filter":{
+         "format":"policy"
```

Comment:
Is `format` distinct from `document_type`? [contacts-admin](https://github.com/alphagov/contacts-admin/blob/master/app/presenters/contacts_finder_presenter.rb#L50) and [specialist-publisher](https://github.com/alphagov/specialist-publisher/blob/master/finders/metadata/cma-cases.json#L7) use `document_type`. If they are describing the same thing, we should standardise.